### PR TITLE
lint: update isort to version >=5

### PR DESCRIPTION
**Pull request by m-col**
re https://github.com/qtile/qtile/issues/1878

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -40,9 +40,9 @@ commands =
 
 [testenv:format]
 deps =
-    isort < 5.0
+    isort >= 5.0
 commands =
-    isort -y -rc {toxinidir}/libqtile {toxinidir}/bin/ {toxinidir}/test
```
Comment (tych0) [inline tox.ini:45]: Does this still exit(1) if there's a failure?

**Comment by m-col** [comment]
It does, after printing the error:

```
TOXENV=format,pep8 tox
format installed: isort==5.5.2
format run-test-pre: PYTHONHASHSEED='793322807'
format run-test: commands[0] | isort /home/mcol/git/qtile/libqtile /home/mcol/git/qtile/bin/ /home/mcol/git/qtile/test
pep8 installed: flake8==3.8.3,flake8-isort==4.0.0,flake8-polyfill==1.0.2,flake8-tidy-imports==4.1.0,isort==5.5.2,mccabe==0.6.1,pep8-naming==0.11.1,pycodestyle==2.6.0,pyflakes==2.2.0,testfixtures==6.14.2
pep8 run-test-pre: PYTHONHASHSEED='793322807'
pep8 run-test: commands[0] | flake8 /home/mcol/git/qtile/libqtile /home/mcol/git/qtile/bin/ /home/mcol/git/qtile/test
libqtile/__init__.py:23:1: F401 'abc' imported but unused
libqtile/__init__.py:24:1: F401 'json' imported but unused
ERROR: InvocationError for command /home/mcol/git/qtile/.tox/pep8/bin/flake8 libqtile bin test (exited with code 1)
_____________________________________________________ summary _____________________________________________________
   format: commands succeeded
ERROR:   pep8: commands failed
make: *** [Makefile:16: lint] Error 1
```

What is the intended outcome?

**Comment by tych0** [comment]
Oh, I see. -rc is recursive, not "return code". So ignore me :)

**Comment by m-col** [comment]
phew!

**Pull request merged**